### DISCUSSION TITLE
NAV POSHOLD JUMPY

### DISCUSSION THREAD

**Discussion by Steve-A15** [iNavFlight/inav#7493] [Q&A]
I have rebuilt my old DJI Phantom I with new electronics Matek F722-SE, BLHeli_32 single ESC's, BN-880  GPS, original 920KV motors, 9" Carbon Fiber Props, Ect. I don't think the hardware is a problem as generally things are working just fine.  I got ALT hold in ANGLE mode working great holding +/- 1 to 2 feet. When I switch to NAV POSHOLD, it holds position just fine but starts to get jumpy. This twitchy behavior seems mostly in the roll axis. I set up filters as per Paweł Spychalski's video at; https://youtu.be/Z74RccOn4Cc and this didn't seem to help at all or very little. I think it's a PID issue but it does not show this twitchy behavior in ANGLE mode just when being guided by the GPS.  I'm not sure what PIDS to start moving or in what direction. Does anyone have any ideas? I have looked at blackbox and I'm not seeing any noise anywhere.

**Comment by lida2003** [comment]
iNav 5.0.0 FW, still seems jumpy when posHold.
